src/qr.rs

```diff
@@ -116,7 +116,7 @@ pub enum Qr {
         /// information to connect to and authenticate a backup provider.
         ///
         /// The format is somewhat opaque, but `sendme` can deserialise this.
-        ticket: iroh::provider::Ticket,
```

Comment:
I actually like the clear distinction between old and new iroh here. We anyway plan to deprecate old iroh in the next months so this will not stay for long.